src/components/homepage/hero-video.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useStaticQuery, graphql } from "gatsby";
+import { Video } from "gatsby-video";
+
+export const PureHeroVideo = ({ data }) => {
+  const videos = data.file.childVideoFfmpeg;
+  return (
+    <Video
+      autoPlay
+      muted
+      loop
+      className="hero-image-wrapper"
+      sources={[videos.webm, videos.mp4]}
+    />
+  );
+};
+
+const HeroVideo = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      file(relativePath: { eq: "homepage-hero.mp4" }) {
+        childVideoFfmpeg {
+          webm: transcode(
+            outputOptions: ["-crf 20", "-b:v 0"]
```

Comment:
might be worth adding a short comment describing what these options actually do

Reply:
tbh I copied and pasted an example so I'll need to look up what it's doing and check it's what we need

Reply:
gonna merge it for now and can loop back to this after the other homepage bits